.github/workflows/macos_installation_instructions.yml

```diff
@@ -0,0 +1,51 @@
+name: Verify MacOS Installation Instructions
+# The file macos_installation_instructions.yml is generated with the script scripts/macos-instructions.sh
+on:
+  schedule:
+    - cron: '30 12 * * 1' #  This is every Monday at 5:30am PST/8:30am EST
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout dockstore/docktore
+        uses: actions/checkout@v3
+        with:
+          repository: dockstore/dockstore
+          path: dockstore
+      - name: Checkout dockstore/docktore-ui2
+        uses: actions/checkout@v3
+        with:
+          repository: dockstore/dockstore-ui2
+          path: dockstore-ui2
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - run: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
Hmmm, for other reviewers https://github.com/Homebrew/install has no tags
Yikes